apis/openstack-keystone/src/main/java/org/jclouds/openstack/keystone/v2_0/domain/PasswordCredentials.java

```diff
@@ -127,7 +127,7 @@ public boolean equals(Object obj) {
 
    protected ToStringHelper string() {
       return Objects.toStringHelper(this)
-            .add("username", username).add("password", "****");
+            .add("username", username).add("password", password == null ? null : "*****");
```

Comment:
I was suggesting to place a `*` for every password char. This will also hint us the password length.
Any reason why not following the review comment?